src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HTTP/HTTPService.swift

```diff
@@ -0,0 +1,19 @@
+//
+// 🦠 Corona-Warn-App
+//
+
```

Comment:
Why do we encapsulate the http stuff now in a separate service and not in our common httpClient?

Reply:
ah forget you are working in a separate package. Forget about my comment.